blockly-core/core/connections/connection.js

```diff
@@ -616,9 +665,18 @@ Blockly.Connection.prototype.setCheck = function(check) {
     if (!(check instanceof Array)) {
       check = [check];
     }
+
     this.check_ = check;
+
+    var legacyTypeFound = Blockly.Connection.findLegacyType_(check);
+    if (legacyTypeFound) {
+      console.log("Legacy Type Found:");
+      console.log(legacyTypeFound);
+      throw 'Error: found legacy type check (see console)';
```

Comment:
Replaced that throw with a replace, on second thought it probably makes sense to keep it as a throw, though, to get alerted about any missed usages sooner rather than later